source/javascripts/components/StepItem/StepItem.tsx

```diff
@@ -65,9 +66,10 @@ const StepItem: FC<StepItemProps> = ({
 		<span className="info">
 			<strong>
 				<Text className="title" ellipsis>
-					{title}
+					{displayName}
 				</Text>
-				{isVerified && <img className="verified" src={verifiedIcon} />}
+				{isVerified && !isDeprecated && <img className="verified" src={verifiedIcon} />}
+				{!isVerified && !isDeprecated && <img className="community-created" src={communityCreatedIcon} />}
```

Comment:
ohh you should catch up with @simonmarton he changed this part and ported it to Bitkit

Reply:
I think it would be better than to let @simonmarton's PR go into master first, and I would rebase my changes to that one.